onyo/cli/tsv_to_yaml.py

```diff
@@ -37,13 +38,15 @@
 
 def tsv_to_yaml(args: argparse.Namespace) -> None:
     r"""
-    Convert a TSV file into YAML suitable to pass to ``onyo new`` and ``onyo set``.
+    Print a TSV file's contents as YAML, suitable to pass into ``onyo new`` and
+    ``onyo set``.
 
-    The header declares the key names to be populated. The values to populate
-    documents are declared with one line per YAML document.
+    The output is printed to stdout as a multi-document YAML file, and each
+    document is separated by a ``---`` line.
```

Comment:
Huge nit, but "separated with" is preferred over "separated by". (I had to look this one up).

"By" is an agent. The men fighting were separated by Horatio.

"With" is for instruments. They were separated with a crowbar. The documents were separated with a `---`.

Reply:
Lol, I love it. It's adjusted!